physx/source/geomutils/src/convex/GuBigConvexData.cpp

```diff
@@ -149,13 +149,13 @@ bool BigConvexData::Load(PxInputStream& stream)
 	// Load base gaussmap
 //	if(!GaussMap::Load(stream))	return false;
 
-		// Import header
-		if(!ReadHeader('G', 'A', 'U', 'S', Version, Mismatch, stream))
-			return false;
+	// Import header
```

Comment:
This file just looks like whitespace changes. Can you revert this? I'd like to limit the changes to existing PhysX code as much to ease ongoing rebasing as well as ease pushing to PhysX once NVidia decides to take this

Reply:
As mentioned in the commit messages, these are "misleading indentation" warnings in clang, that are treaded as errors. I can instead ignore the warning, if you'd prefer that.

Reply:
See https://github.com/prestomation/PhysX/pull/6/commits/c656a9e9a5f4572809311e49a5494625ba72bcf4